### DISCUSSION TITLE
Need a way to run on HTML

### DISCUSSION THREAD

**Discussion by ewarggg776** [UseInterstellar/Interstellar#952] [Q&A] [answered]
Need a way to run on HTML because codespaces and other online ways to run are blocked

**Comment by Charlieis1nsan3** [comment]
bruhhhhh https://www.w3schools.com/html/tryit.asp?filename=tryhtml_basic

**Reply by FRIends367** [reply]
but what code do you put

**Reply by Charlieis1nsan3** [reply]
you cant run interstellar on just a static HTML site -_-

**Reply by cleverman606** [reply]
i mean to be honest eaglercraft is JS so i think you prob could compile the JS into an HTML file but idk

**Comment by Multi-Language-Coder** [comment]
The only way you're getting Interstellar to work is through vercel or GitHub code spaces.

**Comment by xbubbo** [comment, marked as answer]
Interstellar cannot be ran through a html file without heavy modifications and relying on another proxy service to make requests
